models/Course.js

```diff
@@ -0,0 +1,10 @@
+class Course {

```

Comment:
I think this should match the structure of the array of courses you made.

Reply:
It would be helpful to whoever communicates with the backend if the Course model reflected the values we actually want, which would be the ones in the array.

Reply:
That's true, we can discuss more about this in the meeting tonight